governance-charter.md

```diff
@@ -112,19 +122,17 @@ to the respective positions.
 
```

Comment:
```suggestion

In the case of a tie among candidates, which would lead to one candidate being elected and the others being not elected, a (special election)[#special-elections] will be held among those candidates affected by the tie.
```

Sorry for doing that, but while thinking about all things that could go wrong, I ran into the question of ties. Not sure if this should be part of this PR or this should be a follow-up, so feel free to drop this comment and I will do a follow-up, @jpkrohling

Reply:
I think this should be part of a new PR, as it's a new topic.